inlong-audit/audit-store/src/test/resources/application-test.properties

```diff
@@ -16,16 +16,13 @@
 # specific language governing permissions and limitations
 # under the License.
 #
-
```

Comment:
It is recommended to keep these blank lines to make the properties file more readable.

Reply:
ok